patches/@octokit__webhooks-methods@3.0.1.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/package.json b/package.json
+index 7c6e9a1af624be81938679edba36a046fd78f5f9..3dd36bc7573673a082dd8cc4910e0e3aca51c4a9 100644
+--- a/package.json
++++ b/package.json
+@@ -42,6 +42,5 @@
+   },
+   "source": "dist-src/index.js",
+   "types": "dist-types/index.d.ts",
+-  "main": "dist-node/index.js",
+-  "module": "dist-web/index.js"
++  "main": "dist-node/index.js"
```

Comment:
this one is something else? or it's here on purpose?

Reply:
We have two versions

Reply:
indirect dependencies